lib/config/config.js

```diff
@@ -483,12 +491,12 @@ function loadConfigFile (filePath) {
  * @returns {Object} Object with agent, environment, and configuration data.
  */
 function getLoggingPreambleData (config, sources, configFilePath) {
-  // const optsFromSources = sources.reduce((prev, s) => Object.assign({}, s.opts, prev), {})
   const optsFromSources = sources.reduceRight((prev, s) => Object.assign(prev, s.opts), {})
   const result = {
     agentVersion: AGENT_VERSION,
     env: {
       pid: process.pid,
+      tid: threadId, // worker_threads docs do *not* specify if the main thread is always tid 0
```

Comment:
nit: in logging preamble we discussed the size of it. So maybe avoid adding the property if it's undefined would be nice

Reply:
Good point about the size of the preamble. Given that I expect very little worker thread usage, and even less a Worker with the APM agent running in it, this `tid` field will rarely be useful. I'm inclined to not add it for now.